numpy/f2py/tests/test_array_from_pyobj.py

```diff
@@ -154,11 +154,12 @@ def __new__(cls, name):
 
     def _init(self, name):
         self.NAME = name.upper()
+        info = typeinfo[self.NAME]
         self.type_num = getattr(wrap, 'NPY_' + self.NAME)
-        assert_equal(self.type_num, typeinfo[self.NAME][1])
-        self.dtype = typeinfo[self.NAME][-1]
-        self.elsize = typeinfo[self.NAME][2] / 8
-        self.dtypechar = typeinfo[self.NAME][0]
+        assert_equal(self.type_num, info.num)
+        self.dtype = info.type
+        self.elsize = info.bits / 8
```

Comment:
I know it was here already, and it is only a test, so perhaps OK, but it still seems strange not to use `//`.

Reply:
Actually, I'd argue that all of these are wrong, and should be `ceil(bits / 8)` or `must_be_integral(bits / 8)`. Would prefer to leave that out of scope, either way